Failures in [node-test-pull-request/65891](https://ci.nodejs.org/job/node-test-pull-request/65891/) to [node-test-pull-request/65989](https://ci.nodejs.org/job/node-test-pull-request/65989/) that failed 2 or more PRs
(Generated with `ncu-ci walk pr --stats=true --markdown reports/2025-04-03.md`)

| UTC Time         | RUNNING | SUCCESS | UNSTABLE | ABORTED | FAILURE | Green Rate |
| ---------------- | ------- | ------- | -------- | ------- | ------- | ---------- |
| 2025-04-03 00:10 | 0       | 2       | 15       | 4       | 79      | 2.08%      |


### Jenkins Failure

| Reason | <code>Build timed out (after 60 minutes). Marking the build as failed.</code> |
| - | :- |
| Type | JENKINS_FAILURE |
| Failed PR | 11 ([https://github.com/nodejs/node/pull/57686/](https://ci.nodejs.org/job/node-test-pull-request/65922/), [https://github.com/nodejs/node/pull/57653/](https://ci.nodejs.org/job/node-test-pull-request/65930/), [https://github.com/nodejs/node/pull/57613/](https://ci.nodejs.org/job/node-test-pull-request/65934/), [https://github.com/nodejs/node/pull/57596/](https://ci.nodejs.org/job/node-test-pull-request/65935/), [https://github.com/nodejs/node/pull/57589/](https://ci.nodejs.org/job/node-test-pull-request/65937/), [https://github.com/nodejs/node/pull/57421/](https://ci.nodejs.org/job/node-test-pull-request/65940/), [https://github.com/nodejs/node/pull/56778/](https://ci.nodejs.org/job/node-test-pull-request/65941/), [https://github.com/nodejs/node/pull/55752/](https://ci.nodejs.org/job/node-test-pull-request/65942/), [https://github.com/nodejs/node/pull/57585/](https://ci.nodejs.org/job/node-test-pull-request/65943/), [https://github.com/nodejs/node/pull/57704/](https://ci.nodejs.org/job/node-test-pull-request/65973/), [https://github.com/nodejs/node/pull/57640/](https://ci.nodejs.org/job/node-test-pull-request/65976/)) |
| Appeared | [vm-pgk6l](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-x64/64381/console), [vm-7txjc](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-x64/64357/console), [vm-7bflv](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-x64/64356/console), [test-rackspace-win2022_vs2022-x64-6](https://ci.nodejs.org/job/node-compile-windows/nodes=win-vs2022-arm64/61110/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65922/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65976/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-x64/64381/console">Example</a></summary>

```
Build timed out (after 60 minutes). Marking the build as failed.
```
</details>

-------

| Reason | <code>Build timed out (after 30 minutes). Marking the build as failed.</code> |
| - | :- |
| Type | JENKINS_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/57686/](https://ci.nodejs.org/job/node-test-pull-request/65904/), [https://github.com/nodejs/node/pull/57687/](https://ci.nodejs.org/job/node-test-pull-request/65916/)) |
| Appeared | [test-azure_msft-win11_vs2022-arm64-5](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/33299/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65904/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65916/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/33299/console">Example</a></summary>

```
Build timed out (after 30 minutes). Marking the build as failed.
```
</details>

-------


### Git Failure

| Reason | <code>ERROR: Error fetching remote repo 'origin'</code> |
| - | :- |
| Type | GIT_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/57619/](https://ci.nodejs.org/job/node-test-pull-request/65932/), [https://github.com/nodejs/node/pull/57589/](https://ci.nodejs.org/job/node-test-pull-request/65937/)) |
| Appeared | [test-mnx-ubuntu2204-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/63868/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65932/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65937/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/63868/console">Example</a></summary>

```
ERROR: Error fetching remote repo 'origin'
hudson.plugins.git.GitException: Failed to fetch from git@github.com:nodejs/node.git
	at PluginClassLoader for git//hudson.plugins.git.GitSCM.fetchFrom(GitSCM.java:999)
	at PluginClassLoader for git//hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1240)
	at PluginClassLoader for git//hudson.plugins.git.GitSCM._checkout(GitSCM.java:1311)
	at PluginClassLoader for git//hudson.plugins.git.GitSCM.checkout(GitSCM.java:1278)
```
</details>

-------


### Build Failure

| Reason | <code>Error: list index out of range</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 10 ([https://github.com/nodejs/node/pull/57598/](https://ci.nodejs.org/job/node-test-pull-request/65896/), [https://github.com/nodejs/node/pull/57686/](https://ci.nodejs.org/job/node-test-pull-request/65904/), [https://github.com/nodejs/node/pull/57687/](https://ci.nodejs.org/job/node-test-pull-request/65916/), [https://github.com/nodejs/node/pull/57664/](https://ci.nodejs.org/job/node-test-pull-request/65928/), [https://github.com/nodejs/node/pull/57653/](https://ci.nodejs.org/job/node-test-pull-request/65930/), [https://github.com/nodejs/node/pull/57640/](https://ci.nodejs.org/job/node-test-pull-request/65931/), [https://github.com/nodejs/node/pull/57591/](https://ci.nodejs.org/job/node-test-pull-request/65936/), [https://github.com/nodejs/node/pull/52329/](https://ci.nodejs.org/job/node-test-pull-request/65944/), [https://github.com/nodejs/node/pull/57114/](https://ci.nodejs.org/job/node-test-pull-request/65965/), [https://github.com/nodejs/node/pull/57569/](https://ci.nodejs.org/job/node-test-pull-request/65989/)) |
| Appeared | [test-azure_msft-win2016_vs2017-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2016-COMPILED_BY-vs2022_clang/33184/console), [test-rackspace-win2016_vs2015-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2016-COMPILED_BY-vs2022_clang/33184/console), [test-rackspace-win2016_vs2015-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2016-COMPILED_BY-vs2022_clang/33184/console), [test-azure_msft-win2016_vs2017-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2016-COMPILED_BY-vs2022_clang/33184/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65896/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65989/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2016-COMPILED_BY-vs2022_clang/33184/console">Example</a></summary>

```
Error: list index out of range

> if errorlevel 1 exit /b 

> if errorlevel 1 exit /b
```
</details>

-------

| Reason | <code>Failed to trigger node-test-commit</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 7 ([https://github.com/nodejs/node-auto-test/pull/72/](https://ci.nodejs.org/job/node-test-pull-request/65894/), [https://github.com/nodejs/node/pull/57622/](https://ci.nodejs.org/job/node-test-pull-request/65945/), [https://github.com/nodejs/node/pull/57706/](https://ci.nodejs.org/job/node-test-pull-request/65946/), [https://github.com/nodejs/node/pull/57610/](https://ci.nodejs.org/job/node-test-pull-request/65958/), [https://github.com/nodejs/node/pull/57711/](https://ci.nodejs.org/job/node-test-pull-request/65966/), [https://github.com/nodejs/node/pull/57569/](https://ci.nodejs.org/job/node-test-pull-request/65984/), [https://github.com/nodejs/node/pull/57425/](https://ci.nodejs.org/job/node-test-pull-request/65988/)) |
| Appeared | [test-mnx-ubuntu2204-x64-1](https://ci.nodejs.org/job/node-test-pull-request/65988/console), [test-mnx-ubuntu2204-x64-2](https://ci.nodejs.org/job/node-test-pull-request/65952/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65894/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65988/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-pull-request/65988/console">Example</a></summary>

```
Failed to trigger node-test-commit
```
</details>

-------

| Reason | <code>fatal: No rebase in progress?</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 3 ([https://github.com/nodejs/node/pull/57659/](https://ci.nodejs.org/job/node-test-pull-request/65929/), [https://github.com/nodejs/node/pull/57114/](https://ci.nodejs.org/job/node-test-pull-request/65965/), [https://github.com/nodejs/node/pull/57707/](https://ci.nodejs.org/job/node-test-pull-request/65979/)) |
| Appeared | [test-ibm-ubuntu2204_sharedlibs_container-x64-3](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/49802/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-8](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/49789/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-9](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/49766/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-5](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/49755/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65929/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65979/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/49802/console">Example</a></summary>

```
fatal: No rebase in progress?
```
</details>

-------

| Reason | <code>ERROR: Step â€˜Record fingerprints of files to track usageâ€™ failed: no workspace for ...</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/57694/](https://ci.nodejs.org/job/node-test-pull-request/65900/), [https://github.com/nodejs/node/pull/57114/](https://ci.nodejs.org/job/node-test-pull-request/65965/)) |
| Appeared | [test-rackspace-debian12-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=debian12-x64/63892/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65900/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65965/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=debian12-x64/63892/console">Example</a></summary>

```
ERROR: Step â€˜Record fingerprints of files to track usageâ€™ failed: no workspace for node-test-commit-linux/nodes=debian12-x64 #63892
Collecting metadata...
Metadata collection done.
[Checks API] No suitable checks publisher found.
Notifying upstream projects of job completion
Finished: FAILURE
```
</details>

-------

| Reason | <code>error C1085: Cannot write precompiled header file: 'out\Release\obj\cctest\cctest.pch': There is not enough space on the disk. [C:\workspace\node-compile-windows\node\cctest.vcxproj]</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/57551/](https://ci.nodejs.org/job/node-test-pull-request/65948/), [https://github.com/nodejs/node/pull/57592/](https://ci.nodejs.org/job/node-test-pull-request/65951/)) |
| Appeared | [test-rackspace-win2022_vs2022-x64-6](https://ci.nodejs.org/job/node-compile-windows/nodes=win-vs2022-arm64/61139/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65948/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65951/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-compile-windows/nodes=win-vs2022-arm64/61139/console">Example</a></summary>

```
error C1085: Cannot write precompiled header file: 'out\Release\obj\cctest\cctest.pch': There is not enough space on the disk. [C:\workspace\node-compile-windows\node\cctest.vcxproj]

> if errorlevel 1 (
if not defined project_generated echo Building Node with reused solution failed. To regenerate project files use "vcbuild projgen"  
 exit /B 1
```
</details>

-------

| Reason | <code>fatal error: Killed signal terminated program cc1plus</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/57694/](https://ci.nodejs.org/job/node-test-pull-request/65900/), [https://github.com/nodejs/node/pull/57114/](https://ci.nodejs.org/job/node-test-pull-request/65965/)) |
| Appeared | [test-digitalocean-alpine321_container-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=alpine-latest-x64/63892/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-3](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_icu_x64/49789/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-9](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_withoutssl_x64/49789/console), [test-ibm-rhel9_arm_cross_container-x64-1](https://ci.nodejs.org/job/node-cross-compile/nodes=cross-compiler-rhel9-armv7-gcc-12-glibc-2.28/52713/console), [test-digitalocean-rhel9_arm_cross_container-x64-1](https://ci.nodejs.org/job/node-cross-compile/nodes=cross-compiler-rhel9-armv7-gcc-12-glibc-2.28/52677/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65900/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65965/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=alpine-latest-x64/63892/console">Example</a></summary>

```
fatal error: Killed signal terminated program cc1plus
```
</details>

-------


### JSTest Failure

| Reason | <code>pummel/test-buffer-large-size</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 23 ([https://github.com/nodejs/node/pull/57598/](https://ci.nodejs.org/job/node-test-pull-request/65895/), [https://github.com/nodejs/node/pull/57687/](https://ci.nodejs.org/job/node-test-pull-request/65899/), [https://github.com/nodejs/node/pull/57575/](https://ci.nodejs.org/job/node-test-pull-request/65912/), [https://github.com/nodejs/node/pull/57599/](https://ci.nodejs.org/job/node-test-pull-request/65919/), [https://github.com/nodejs/node/pull/57683/](https://ci.nodejs.org/job/node-test-pull-request/65924/), [https://github.com/nodejs/node/pull/57682/](https://ci.nodejs.org/job/node-test-pull-request/65925/), [https://github.com/nodejs/node/pull/57664/](https://ci.nodejs.org/job/node-test-pull-request/65928/), [https://github.com/nodejs/node/pull/57613/](https://ci.nodejs.org/job/node-test-pull-request/65934/), [https://github.com/nodejs/node/pull/57596/](https://ci.nodejs.org/job/node-test-pull-request/65935/), [https://github.com/nodejs/node/pull/57591/](https://ci.nodejs.org/job/node-test-pull-request/65936/), [https://github.com/nodejs/node/pull/57589/](https://ci.nodejs.org/job/node-test-pull-request/65937/), [https://github.com/nodejs/node/pull/56778/](https://ci.nodejs.org/job/node-test-pull-request/65941/), [https://github.com/nodejs/node/pull/55752/](https://ci.nodejs.org/job/node-test-pull-request/65942/), [https://github.com/nodejs/node/pull/52329/](https://ci.nodejs.org/job/node-test-pull-request/65944/), [https://github.com/nodejs/node/pull/57592/](https://ci.nodejs.org/job/node-test-pull-request/65951/), [https://github.com/nodejs/node/pull/57622/](https://ci.nodejs.org/job/node-test-pull-request/65961/), [https://github.com/nodejs/node/pull/57619/](https://ci.nodejs.org/job/node-test-pull-request/65963/), [https://github.com/nodejs/node/pull/57686/](https://ci.nodejs.org/job/node-test-pull-request/65964/), [https://github.com/nodejs/node/pull/57711/](https://ci.nodejs.org/job/node-test-pull-request/65967/), [https://github.com/nodejs/node/pull/57707/](https://ci.nodejs.org/job/node-test-pull-request/65968/), [https://github.com/nodejs/node/pull/57659/](https://ci.nodejs.org/job/node-test-pull-request/65974/), [https://github.com/nodejs/node/pull/57706/](https://ci.nodejs.org/job/node-test-pull-request/65981/), [https://github.com/nodejs/node/pull/57479/](https://ci.nodejs.org/job/node-test-pull-request/65986/)) |
| Appeared | [test-ibm-ubi81_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubi81_sharedlibs_openssl111fips_x64/49808/console), [test-ibm-ubuntu2204_sharedlibs_container-x64-2](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_zlib_x64/49808/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-4](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_icu_x64/49797/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-7](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_icu_x64/49791/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_smallicu_x64/49790/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-8](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl32_x64/49787/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-5](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl30_x64/49786/console), [test-ibm-ubuntu2204_sharedlibs_container-x64-3](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl32_x64/49783/console), [test-ibm-ubuntu2204_sharedlibs_container-x64-4](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_smallicu_x64/49775/console), [test-ibm-ubuntu2204_sharedlibs_container-x64-5](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl111_x64/49763/console), [test-digitalocean-ubi81_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubi81_sharedlibs_openssl111fips_x64/49762/console), [test-ibm-ubuntu2204_sharedlibs_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_zlib_x64/49762/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-2](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl111_x64/49760/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-10](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_smallicu_x64/49760/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-3](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl111_x64/49750/console), [test-digitalocean-alpine321_container-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=alpine-latest-x64/63829/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65895/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65986/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubi81_sharedlibs_openssl111fips_x64/49808/console">Example</a></summary>

```
not ok 4036 pummel/test-buffer-large-size
  ---
  duration_ms: 720038.79200
  severity: fail
  exitcode: -15
  stack: |-
    timeout
    (node:151266) [DEP0030] DeprecationWarning: SlowBuffer() is deprecated. Please use Buffer.allocUnsafeSlow()
    (Use `node --trace-deprecation ...` to show where the warning was created)
  ...

```
</details>

-------

| Reason | <code>parallel/test-file-write-stream4</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 10 ([https://github.com/nodejs/node/pull/57598/](https://ci.nodejs.org/job/node-test-pull-request/65895/), [https://github.com/nodejs/node/pull/57438/](https://ci.nodejs.org/job/node-test-pull-request/65905/), [https://github.com/nodejs/node/pull/57599/](https://ci.nodejs.org/job/node-test-pull-request/65906/), [https://github.com/nodejs/node/pull/57687/](https://ci.nodejs.org/job/node-test-pull-request/65910/), [https://github.com/nodejs/node/pull/57664/](https://ci.nodejs.org/job/node-test-pull-request/65928/), [https://github.com/nodejs/node/pull/57585/](https://ci.nodejs.org/job/node-test-pull-request/65943/), [https://github.com/nodejs/node/pull/57622/](https://ci.nodejs.org/job/node-test-pull-request/65961/), [https://github.com/nodejs/node/pull/57479/](https://ci.nodejs.org/job/node-test-pull-request/65977/), [https://github.com/nodejs/node/pull/57627/](https://ci.nodejs.org/job/node-test-pull-request/65982/), [https://github.com/nodejs/node/pull/57610/](https://ci.nodejs.org/job/node-test-pull-request/65983/)) |
| Appeared | [vm-bpvg5](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-arm64/64385/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-7](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_smallicu_x64/49805/console), [test-equinix-rhel8_container-arm64-1](https://ci.nodejs.org/job/node-test-commit-arm/nodes=rhel8-arm64/57888/console), [vm-7j6z9](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-arm64/64370/console), [test-osuosl-ubuntu2204_container-armv7l-1](https://ci.nodejs.org/job/node-test-commit-arm/nodes=ubuntu2204-armv7l/57865/console), [test-azure_msft-win10_vs2019-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/33312/console), [vm-8mn2w](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-arm64/64326/console), [test-equinix-rhel8_container-arm64-2](https://ci.nodejs.org/job/node-test-commit-arm/nodes=rhel8-arm64/57830/console), [test-digitalocean-fedora40-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/63831/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65895/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65983/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx13-arm64/64385/console">Example</a></summary>

```
not ok 4031 parallel/test-file-write-stream4
  ---
  duration_ms: 360090.38500
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>sequential/test-tls-session-timeout</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 6 ([https://github.com/nodejs/node/pull/57598/](https://ci.nodejs.org/job/node-test-pull-request/65895/), [https://github.com/nodejs/node/pull/57438/](https://ci.nodejs.org/job/node-test-pull-request/65905/), [https://github.com/nodejs/node/pull/57659/](https://ci.nodejs.org/job/node-test-pull-request/65929/), [https://github.com/nodejs/node/pull/57591/](https://ci.nodejs.org/job/node-test-pull-request/65936/), [https://github.com/nodejs/node/pull/57693/](https://ci.nodejs.org/job/node-test-pull-request/65955/), [https://github.com/nodejs/node/pull/57707/](https://ci.nodejs.org/job/node-test-pull-request/65979/)) |
| Appeared | [test-digitalocean-ubuntu2204_sharedlibs_container-x64-6](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl35_x64/49802/console), [test-ibm-ubuntu2204_sharedlibs_container-x64-5](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl35_x64/49778/console), [test-azure_msft-win10_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2022/33321/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-10](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl32_x64/49755/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65895/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65979/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl35_x64/49802/console">Example</a></summary>

```
not ok 4457 sequential/test-tls-session-timeout
  ---
  duration_ms: 168.76400
  severity: fail
  exitcode: 1
  stack: |-
    node:assert:95
      throw new AssertionError(obj);
      ^
    
    AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
    
    'New' !== 'Reused'
    
        at /home/iojs/build/workspace/node-test-commit-linux-containered/test/sequential/test-tls-session-timeout.js:150:16
        at ChildProcess.<anonymous> (/home/iojs/build/workspace/node-test-commit-linux-containered/test/sequential/test-tls-session-timeout.js:134:7)
        at ChildProcess.emit (node:events:507:28)
        at ChildProcess._handle.onexit (node:internal/child_process:294:12) {
      generatedMessage: true,
      code: 'ERR_ASSERTION',
      actual: 'New',
      expected: 'Reused',
      operator: 'strictEqual'
    }
    
    Node.js v24.0.0-pre
  ...

```
</details>

-------

| Reason | <code>pummel/test-heapdump-http2</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 4 ([https://github.com/nodejs/node/pull/57695/](https://ci.nodejs.org/job/node-test-pull-request/65926/), [https://github.com/nodejs/node/pull/57676/](https://ci.nodejs.org/job/node-test-pull-request/65927/), [https://github.com/nodejs/node/pull/57664/](https://ci.nodejs.org/job/node-test-pull-request/65928/), [https://github.com/nodejs/node/pull/57610/](https://ci.nodejs.org/job/node-test-pull-request/65959/)) |
| Appeared | [test-equinix-ubuntu2204_container-arm64-5](https://ci.nodejs.org/job/node-test-commit-arm-debug/nodes=ubuntu2204_debug-arm64/17852/console), [test-rackspace-debian12-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=debian12-x64/63857/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65926/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65959/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-arm-debug/nodes=ubuntu2204_debug-arm64/17852/console">Example</a></summary>

```
not ok 4048 pummel/test-heapdump-http2
  ---
  duration_ms: 3515.92900
  severity: crashed
  exitcode: -5
  stack: |-
    (node:3053532) internal/test/binding: These APIs are for internal testing only. Do not use them.
    (Use `node --trace-warnings ...` to show where the warning was created)
    
    
    #
    # Fatal error in ../deps/v8/src/profiler/heap-snapshot-generator.cc, line 625
    # Debug check failed: is_native_object_bool == entry_info.id % 2 == 0 (1 vs. 0).
    #
    #
    #
    #FailureMessage Object: 0xffffcf0a0fa8
    ----- Native stack trace -----
    
     1: 0xaaaae25e1694 node::DumpNativeBacktrace(_IO_FILE*) [out/Debug/node]
     2: 0xaaaae284f3f0  [out/Debug/node]
     3: 0xaaaae284f424  [out/Debug/node]
     4: 0xaaaae4ee6c60 V8_Fatal(char const*, int, char const*, ...) [out/Debug/node]
     5: 0xaaaae4ee6ca8  [out/Debug/node]
     6: 0xaaaae36c10a8 v8::internal::HeapObjectsMap::FindOrAddEntry(unsigned long, unsigned int, v8::internal::HeapObjectsMap::MarkEntryAccessed, v8::internal::...
```
</details>

-------

| Reason | <code>es-module/test-typescript-eval</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/57598/](https://ci.nodejs.org/job/node-test-pull-request/65896/), [https://github.com/nodejs/node/pull/57687/](https://ci.nodejs.org/job/node-test-pull-request/65903/)) |
| Appeared | [test-azure_msft-win10_vs2019-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/33281/console), [test-azure_msft-win10_vs2019-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022_clang/33281/console), [test-azure_msft-win11_vs2022-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2022/33281/console), [test-azure_msft-win11_vs2022-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2022_clang/33281/console), [test-azure_msft-win11_vs2022-arm64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/33281/console), [test-azure_msft-win11_vs2022-arm64-5](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022_clang-arm64/33281/console), [test-rackspace-win2019_vs2019-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2019-COMPILED_BY-vs2022/33281/console), [test-rackspace-win2022_vs2022-x64-5](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2022-COMPILED_BY-vs2022/33281/console), [test-azure_msft-win10_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022_clang/33279/console), [test-azure_msft-win11_vs2022-arm64-6](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/33279/console), [test-rackspace-win2019_vs2019-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2019-COMPILED_BY-vs2022/33279/console), [test-rackspace-win2022_vs2022-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2022-COMPILED_BY-vs2022/33279/console), [test-azure_msft-win11_vs2022-x64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2022/33277/console), [test-azure_msft-win11_vs2022-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2022/33275/console), [test-rackspace-win2019_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2019-COMPILED_BY-vs2022/33275/console), [test-azure_msft-win11_vs2022-arm64-2](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-arm64-COMPILED_BY-vs2022-arm64/33266/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65896/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65903/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/33281/console">Example</a></summary>

```
not ok 95 es-module/test-typescript-eval
  ---
  duration_ms: 3176.00500
  severity: fail
  exitcode: 1
  stack: |-
    Test failure: 'the error message should not contain extra padding'
    Location: test\es-module\test-typescript-eval.mjs:268:1
    AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
    + actual - expected
    
    + '[eval]:1   \r'
    - '[eval]:1   '
                  ^
    
        at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/es-module/test-typescript-eval.mjs:276:3)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async Test.run (node:internal/test_runner/test:1063:7)
        at async Test.processPendingSubtests (node:internal/test_runner/test:746:7) {
      generatedMessage: true,
      code: 'ERR_ASSERTION',
      actual: '[eval]:1   \r',
      expected: '[eval]:1   ',
      operator: 'strictEqual'
    }
    
  ...

```
</details>

-------

| Reason | <code>parallel/test-cluster-fork-stdio</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/57687/](https://ci.nodejs.org/job/node-test-pull-request/65903/), [https://github.com/nodejs/node/pull/57686/](https://ci.nodejs.org/job/node-test-pull-request/65904/)) |
| Appeared | [test-azure_msft-win11_vs2022-arm64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2019-arm64/33274/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65903/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65904/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-arm64-COMPILED_BY-vs2019-arm64/33274/console">Example</a></summary>

```
not ok 967 parallel/test-cluster-fork-stdio
  ---
  duration_ms: 120164.40200
  severity: fail
  exitcode: 1
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>parallel/test-http-chunk-problem</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/57694/](https://ci.nodejs.org/job/node-test-pull-request/65900/), [https://github.com/nodejs/node/pull/57479/](https://ci.nodejs.org/job/node-test-pull-request/65977/)) |
| Appeared | [test-digitalocean-ubuntu2204_sharedlibs_container-x64-7](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/49800/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-5](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl31_x64/49725/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65900/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65977/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/49800/console">Example</a></summary>

```
not ok 1503 parallel/test-http-chunk-problem
  ---
  duration_ms: 120028.06300
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>pummel/test-timers</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/52329/](https://ci.nodejs.org/job/node-test-pull-request/65944/), [https://github.com/nodejs/node/pull/57659/](https://ci.nodejs.org/job/node-test-pull-request/65974/)) |
| Appeared | [test-digitalocean-ubi81_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubi81_sharedlibs_openssl111fips_x64/49797/console), [test-azure_msft-win10_vs2019-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/33329/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/65944/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/65974/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubi81_sharedlibs_openssl111fips_x64/49797/console">Example</a></summary>

```
not ok 4082 pummel/test-timers
  ---
  duration_ms: 1462.36400
  severity: fail
  exitcode: 1
  stack: |-
    diff: 1001
    diff: 1272
    node:internal/assert/utils:281
        throw err;
        ^
    
    AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:
    
      assert.ok(Math.abs(diff - t) < WINDOW * interval_count)
    
        at Timeout.<anonymous> (/home/iojs/build/workspace/node-test-commit-linux-containered/test/pummel/test-timers.js:64:12)
        at Timeout._onTimeout (/home/iojs/build/workspace/node-test-commit-linux-containered/test/common/index.js:435:15)
        at listOnTimeout (node:internal/timers:608:17)
        at process.processTimers (node:internal/timers:543:7) {
      generatedMessage: true,
      code: 'ERR_ASSERTION',
      actual: false,
      expected: true,
      operator: '=='
    }
    
    Node.js v24.0.0-pre
  ...

```
</details>

-------

